File: app/src/main/java/com/widget/testhsmkt/support/ObjectEvent.java
package com.widget.testhsmkt.support;

public class ObjectEvent
{
	private int		event_	= 0x00;
	private Object	data_	= null;
	
	public ObjectEvent(int event, Object data)
	{
		event_	= event;
		data_	= data;
	}

	public	int Event()
	{
		return	event_;
	}

	public	Object Data()
	{
		return	data_;
	}
}

File: app/src/main/java/com/widget/testhsmkt/support/ObjectEvent.kt
package com.widget.testhsmkt.support

class ObjectEvent
    (event: Int, data: Any?) {
    private var event_ = 0x00
    private var data_: Any? = null

    init {
        event_ = event
        data_ = data
    }

    fun event(): Int {
        return event_
    }

    fun data(): Any? {
        return data_
    }
}
